src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -101,7 +101,7 @@ public async Task<ActionResult> LogInTestUser(StartAppModel startAppModel)
 
             List<Claim> claims = new List<Claim>();
             string issuer = "altinn3local.no";
-            claims.Add(new Claim("sub", profile.UserId.ToString(), ClaimValueTypes.String, issuer));
+            claims.Add(new Claim(ClaimTypes.NameIdentifier, profile.UserId.ToString(), ClaimValueTypes.String, issuer));
```

Comment:
this isn't really used anywhere else in the code yet, right?

Reply:
Correct. It is just used by asp.net framework